.github/workflows/ondemand.yml

```diff
@@ -0,0 +1,45 @@
+name: Regression on demand tests
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    name: Cypress Regression on demand tests
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1, 2, 3, 4, 5]
```

Comment:
Why this change? From what I see each container still runs for about 7 minutes. There's more overhead starting the machine than running the tests themselves.